src/apps/media/src/app/components/MediaHeader.js

```diff
@@ -10,21 +10,34 @@ import { MediaCreateGroupModal } from "./MediaCreateGroupModal";
 import { MediaEditGroupModal } from "./MediaEditGroupModal";
 
 import styles from "./MediaHeader.less";
+import UploadImage from "./UploadImage";
 
 export const MediaHeader = memo(function MediaHeader(props) {
   const userRole = useSelector((state) => state.userRole);
   const [createGroupModal, setCreateGroupModal] = useState(false);
   const [editGroupModal, setEditGroupModal] = useState(false);
 
   return (
-    <header className={styles.WorkspaceHeader}>
+    <header
+      className={cx(
+        styles.WorkspaceHeader,
+        props.groupID ? styles.WorkSpaceHeaderUpload : ""
+      )}
+    >
       <h1 className={cx(styles.subheadline, styles.Title)}>
         <small className={styles.NumFiles}>({props.numFiles})</small>
         {props.searchTerm
           ? `Search Results "${props.searchTerm}"`
           : props.currentGroup.name}
       </h1>
 
+      {props.groupID && (
+        <UploadImage
+          currentGroup={props.currentGroup}
```

Comment:
I think naming the prop for this component currentX is misleading. Since its a higher level abstraction that can upload anything without it needing context of whether or not it is 'current' I think a much better name for it is simply 'group' and 'bin'.